pkg/provision/zdb.go

```diff
@@ -193,6 +193,25 @@ func createZdbContainer(ctx context.Context, allocation pkg.Allocation, mode pkg
 		return errors.Wrap(err, "failed to create container")
 	}
 
+	socket := fmt.Sprintf("unix://%s/zdb.sock", socketDir)
+	cl := zdb.New(socket)
+	defer cl.Close()
+
+	check := func() error {
```

Comment:
you don't need to wrap it here. just use `cl.Connect` in the `RetryNotify`

Reply:
Yup, true. :stuck_out_tongue_winking_eye: